copier/cli.py

```diff
@@ -144,7 +144,7 @@ class _Subcommand(cli.Application):
         help="Use prereleases to compare template VCS tags.",
     )
     unsafe = cli.Flag(
-        ["--UNSAFE"],
+        ["--UNSAFE", "--trusted"],
```

Comment:
Or `--trust` instead of `--trusted`? :thinking: "Unsafe" is an adjective, "trusted" as well, but many other switches/flags are in imperative mood. Not sure. WDYT, @yajo @pawamoy?

Reply:
+1 for `trust`. Shorter, easier to write and pronounce. It better conveys the fact that you want to trust the template.

Reply:
Thanks! Done.